app/scripts/abos/list/aboslist.html

```diff
@@ -29,6 +29,14 @@ <h3 class="panel-title"><span class="navbar-brand"><span class="small">{{abo.id}
              {{abo.vertrieb.beschrieb| translate}}
           </div>
         </div>
+        <div class="row vertrieb" ng-if="abo.vertrieb.beschrieb">
```

Comment:
can you replace the css class `vertrieb` by `abotyp`? Attributing different classes to each sections makes it possible to hide sections using custom CSS easily.

Reply:
done